lib/has_content_id.rb

```diff
@@ -0,0 +1,14 @@
+module HasContentId
```

Comment:
I feel like this name could better describe the behaviour the including thing would exhibit. Maybe something like `ContentIdentifiable`?